From 68e94b7c0d8f939c6f517e322ad5864a15310e1b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sat, 24 Oct 2026 19:11:22 +0200
Subject: [PATCH] Use 'edit-clear-all' icon for Clear Recent Documents

This menu item clears the recent documents list, not just one item.
Replace the icon 'edit-clear' with 'edit-clear-all'.
---
 mate-panel/panel-recent.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mate-panel/panel-recent.c b/mate-panel/panel-recent.c
index 5fe25ddb4..aa2ccabe0 100644
--- a/mate-panel/panel-recent.c
+++ b/mate-panel/panel-recent.c
@@ -157,7 +157,7 @@ recent_documents_clear_cb (GtkMenuItem      *menuitem,
 				 GTK_RESPONSE_CANCEL);
 
 	panel_dialog_add_button (GTK_DIALOG (clear_recent_dialog),
-				 _("_Clear"), "edit-clear",
+				 _("_Clear"), "edit-clear-all",
 				 GTK_RESPONSE_ACCEPT);
 
 	gtk_container_set_border_width (GTK_CONTAINER (clear_recent_dialog), 6);
@@ -240,7 +240,7 @@ panel_recent_append_documents_menu (GtkWidget        *top_menu,
 	setup_menuitem_with_icon (menu_item,
 				   panel_menu_icon_get_size (),
 				   NULL,
-				   "edit-clear",
+				   "edit-clear-all",
 				   _("Clear Recent Documents..."));
 	panel_util_set_tooltip_text (menu_item,
 				     _("Clear all items from the recent documents list"));
